addons/strings/stringtable.xml

```diff
@@ -30,13 +31,8 @@
         </Key>
         <Key ID="STR_CBA_FORMAT_NUMBER_THOUSANDS_SEPARATOR">
             <English>,</English>
-            <Czech> </Czech>
```

Comment:
don't remove these

Reply:
Does arma support non breaking space character? I wonder if that could be used here, it's really easy to mess that when using Tabler.

Reply:
How would I differentiate between a user error and intentional blank space :/